lib/conditionals.md

```diff
@@ -5,8 +5,22 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What is a conditional?
 
-Replace me with your response...
+A conditional in Ruby is like saying "If...then...". It creates a condition that must be met before the second part of the statement can occur.
+A conditional is always either true or false. A true condition will signify the start of an operation. A false one will not do anything.
+In Ruby 'if' is used to start a conditional and a "conditional operator" like '==' or '>='. If that is met the loop starts.
+If not the conditional can stop or branch off using 'elseif' if a certain other condition is met or 'else' when no conditions are met.
+
 
 # What is the difference between `=` and `==`?
 
-Replace me with your response...
\ No newline at end of file
+A singular '=' is used to define a variable, similar to its use in mathematics. 
+Starting off by defining 'a = 100' will allow the programmer to use 'a' instead of typing out '100' when necessary. 
+In that example the variable name is 'a' and its value is 100.
+
+The double equal sign '==' is used in conditional loops and is one of several conditional operators. 
+I understand them as being a part of the 'if' in an if-then statement. 
+They are always either 'true' or 'false' and that condition will signify whether the loop will proceed, branch off, or stop altogether. 
```

Comment:
To pare the explanation back just a little bit more, where a singular `=` is an _assignment_ operator, `==` is a _comparison_ operator that -- as you said -- returns `true` or `false`